README.md

```diff
@@ -0,0 +1,33 @@
+# RFCs for the Cloud Native Buildpacks project
+
+Want to suggest a change to the Cloud Native Buildpacks project? Awesome!
+
+We follow an RFC (Request for Comments) process for substantial changes to the project.
+This process was originally described in the [RFC RFC](./text/0004-rfc-process.md) and is copied below:
+
+## RFC Process
+
+To get an RFC into Cloud Native Buildpacks, first the RFC needs to be merged into the RFC repo. Once an RFC is merged, it's considered 'active' and may be implemented to be included in the project. These steps will get an RFC to be considered:
```

Comment:
Lots of RFCs. Would this make sense?

```suggestion
To get a proposal into Cloud Native Buildpacks, first, an RFC needs to be merged into the RFC repo. Once an RFC is merged, it's considered 'active' and may be implemented to be included in the project. These steps will get an RFC to be considered:
```

Reply:
This language is copied directly from RFC#4, so unless we're amending that I think it should just be a direct copy.

Reply:
I think it's also ok to update english/wording without needing to touch the RFC.